tests/fixtures/civic_aid6.yaml

```diff
@@ -0,0 +1,539 @@
+      - name: Variant types
+        value:
+          - code: SO:0001583
+            label: missense_variant
+            system: http://www.sequenceontology.org/browser/current_svn/term/
+    id: civic.mpid:33
+    label: EGFR L858R
+    mappings:
+      - coding:
+          code: "33"
+          system: https://civicdb.org/variants/
+        relation: exactMatch
+      - coding:
+          code: CA126713
+          system: https://reg.clinicalgenome.org/
+        relation: relatedMatch
+      - coding:
+          code: "16609"
+          system: https://www.ncbi.nlm.nih.gov/clinvar/variation/
+        relation: relatedMatch
+      - coding:
+          code: "376282"
+          system: https://www.ncbi.nlm.nih.gov/clinvar/variation/
+        relation: relatedMatch
+      - coding:
+          code: "376280"
+          system: https://www.ncbi.nlm.nih.gov/clinvar/variation/
+        relation: relatedMatch
+      - coding:
+          code: rs121434568
+          system: https://www.ncbi.nlm.nih.gov/snp/
+        relation: relatedMatch
+    members:
+      - digest: pM_eD8ha-bnAu6wJOoQTtHYIvEShSN51
+        expressions:
+          - syntax: hgvs.g
+            value: NC_000007.13:g.55259515T>G
+        extensions:
+          - name: mane_genes
+            value:
+              - hgnc_id: 3236
+                ncbi_gene_id: 1956
+                symbol: EGFR
+        id: ga4gh:VA.pM_eD8ha-bnAu6wJOoQTtHYIvEShSN51
+        label: NC_000007.13:g.55259515T>G
+        location:
+          digest: 7g6PIIHJ_QkKe_dRvkuCe8UtZCmPxo5B
+          end: 55191822
+          id: ga4gh:SL.7g6PIIHJ_QkKe_dRvkuCe8UtZCmPxo5B
+          sequence: T
+          sequenceReference:
+            refgetAccession: SQ.F-LrLMe1SRpfUZHkQmvkVKFEGaoDeHul
+            type: SequenceReference
+          start: 55191821
+          type: SequenceLocation
+        state:
+          sequence: G
+          type: LiteralSequenceExpression
+        type: Allele
+      - digest: gV7_dnvF8SQSeUdvgDFhU65zK_csc6VE
+        expressions:
+          - syntax: hgvs.c
+            value: NM_005228.4:c.2573T>G
+        id: ga4gh:VA.gV7_dnvF8SQSeUdvgDFhU65zK_csc6VE
+        label: NM_005228.4:c.2573T>G
+        location:
+          digest: LREsUiEYvOrRhwXW1rG72kXFPegvkNzI
+          end: 2834
+          id: ga4gh:SL.LREsUiEYvOrRhwXW1rG72kXFPegvkNzI
+          sequence: T
+          sequenceReference:
+            refgetAccession: SQ.d_QsP29RWJi6bac7GOC9cJ9AO7s_HUMN
+            type: SequenceReference
+          start: 2833
+          type: SequenceLocation
+        state:
+          sequence: G
+          type: LiteralSequenceExpression
+        type: Allele
+    type: CategoricalVariant
+  type: VariantTherapeuticResponseProposition
+direction: supports
+classification:
+  conceptType: Guideline
+  mappings:
+  - coding:
+      code: IA
+      system: AMP/ASCO/CAP
+      systemVersion: "2017"
+    relation: exactMatch
+  primaryCode: IA
+hasEvidenceLines:
+  - directionOfEvidenceProvided: supports
```

Comment:
@ahwagner This is a required property. I think for CIViC, it would always be `supports`?